### PR TITLE
[incubator-kie-drools#5710] [new-parser] Unable to resolve method (fr…

### PR DESCRIPTION
**Pull request by tkobayas**
…om new) in a query

**Issue**: 

* https://github.com/apache/incubator-kie-drools/issues/5710

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- for <b>pull request and downstream checks</b>  
  - Push a new commit to the PR. An empty commit would be enough.

- for a <b>full downstream build</b>
  - for <b>github actions</b> job: add the label `run_fdb`

- for <b>Jenkins PR check only</b>
  - If you are an ASF committer for KIE podling, login to Jenkins (https://ci-builds.apache.org/job/KIE/job/drools/), go to the specific PR job, and click on `Build Now` button.
</details>

### CODE REVIEW

**Comment by tkobayas** [comment]
`org.drools.model.codegen.execmodel.QueryTest#testQueryWithDyanmicInsert` and `org.drools.model.codegen.execmodel.FromTest#testNew` are fixed.

Before PR
`drools-model-codegen`
```
[ERROR] Tests run: 2433, Failures: 336, Errors: 0, Skipped: 9
```

After PR
`drools-model-codegen`
```
[ERROR] Tests run: 2433, Failures: 328, Errors: 0, Skipped: 9
```

File: drools-drl/drools-drl-parser/src/main/java/org/drools/drl/parser/antlr4/DRLVisitorImpl.java [hunk 1]
```diff
@@ -543,7 +543,7 @@ public BehaviorDescr visitPatternFilter(DRLParser.PatternFilterContext ctx) {
     @Override
     public FromDescr visitFromExpression(DRLParser.FromExpressionContext ctx) {
         FromDescr fromDescr = new FromDescr();
-        fromDescr.setDataSource(new MVELExprDescr(ctx.getText()));
+        fromDescr.setDataSource(new MVELExprDescr(getTextPreservingWhitespace(ctx)));
```
Comment (tkobayas) [inline drools-drl/drools-drl-parser/src/main/java/org/drools/drl/parser/antlr4/DRLVisitorImpl.java:551]: The previous code didn't preserve a white-space, so resulted in `newPerson`.

**Comment by tkobayas** [comment]
@yurloc @mariofusco @gitgabrio Please review, thanks!

**Review by mariofusco — APPROVED** [review]

**Review by yurloc — APPROVED** [review]
Nice!

**Comment by tkobayas** [comment]
Rebased to resolve a conflict.

**Comment by yurloc** [comment]
Overall fixing progress: There were 364 failed tests.

**Pull request merged**